crates/ruff_linter/src/rules/flake8_pytest_style/types.rs

```diff
@@ -24,7 +24,7 @@ impl Default for ParametrizeNameType {
 impl Display for ParametrizeNameType {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         match self {
-            Self::Csv => write!(f, "csv"),
+            Self::Csv => write!(f, "string of comma-separated values"),
             Self::Tuple => write!(f, "tuple"),
             Self::List => write!(f, "list"),
```

Comment:
...And I'd add backticks here for the two enum variants where it specifically makes sense to add backticks:

```suggestion
            Self::Tuple => write!(f, "`tuple`"),
            Self::List => write!(f, "`list`"),
```

Reply:
Done. I couldn't do this exactly, because the value was used somewhere else where it wouldn't make sense.